MekHQ/src/mekhq/campaign/ResolveScenarioTracker.java

```diff
@@ -375,7 +412,7 @@ private UnitStatus processAlliedUnit(Entity e) {
         UnitStatus us = new UnitStatus(nu);
         unitsStatus.put(nu.getId(), us);
         alliedUnits.add(nu);
-
+        
```

Comment:
```suggestion

```

Reply:
Fixed. My IDE puts this space in when I add a new line. I think I fixed that in preferences from here forward.